src/Refactoring-Core/RBReplacePreview.class.st

```diff
@@ -100,7 +100,7 @@ RBReplacePreview >> initializeWidgets [
 
 { #category : #'instance creation' }
 RBReplacePreview >> open [
-	^ self "openDialogWithSpec"openModalWithSpec
+	^ self "openDialogWithSpec"openModal
```

Comment:
should the comment not also have "openDialog" instead of "openDialogWithSpec" ?

Reply:
yes but more probably, the comment should be removed